default/python/AI/InvasionAI.py

```diff
@@ -367,7 +375,8 @@ def evaluate_invasion_planet(planet_id, empire, secure_fleet_missions, verbose=T
         troop_cost = math.ceil(planned_troops/6.0) * (40*(1+foAI.foAIstate.shipCount * AIDependencies.SHIP_UPKEEP))
     else:
         troop_cost = math.ceil(planned_troops/6.0) * (20*(1+foAI.foAIstate.shipCount * AIDependencies.SHIP_UPKEEP))
-    planet_score = retaliation_risk_factor(planet.owner) * threat_factor * max(0, pop_val+supply_val+bld_tally+enemy_val-0.8*troop_cost)
+    base_score = pop_val + supply_val + bld_tally + tech_tally + enemy_val - 0.8*troop_cost
```

Comment:
Did I missed or there is not logical changes?   Please extract it as separate commit and merge it to master.  No need to mix feature and cleanups.